Switch to noarch python

jakirkham opened the pull request:
Build this package as a noarch Python package since it is pure Python, Python 2/3 compatible, and has no OS specific requirements.

jakirkham committed:
Make package a noarch Python package

This is a perfect candidate for a noarch Python package as it is Python
2/3 compatible and has no Python or OS specific requirements. So switch
it over to a noarch Python package to cutdown on the CI resources it
uses and make it easier to maintain.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jakirkham committed:
MNT: Re-rendered with conda-smithy 2.4.0

After making this a noarch Python package, re-render it so that only
CircleCI will be used to build the package once for all Python versions.

jakirkham committed:
Add spaces around the entry point's `=`

Seems that `conda-build` is sensitive to having spaces around the `=`s.
Otherwise it mishandles this when building a `noarch: python` package.
Adding spaces seems to resolve the issue.

jakirkham committed:
Bump build number to 1

Needed as we are making a noarch Python package now.

jakirkham commented on recipe/meta.yaml:
```diff
@@ -11,11 +11,12 @@ source:
   preserve_egg_dir: True
   entry_points:
-    - statuspage=statuspage.statuspage:cli
+    - statuspage = statuspage.statuspage:cli
```
Needed to add spaces around `=` because of issue ( https://github.com/conda/conda-build/issues/2347 ).
